modules/Link.js

```diff
@@ -20,6 +20,21 @@ function isEmptyObject(object) {
   return true
 }
 
+function isLinkActive(router, props) {
```

Comment:
Doesn't this make the link always re-render twice on location updates then? Once with the old active state, once with the recalculated active state?